### PR TITLE
Added null string field info to Working with strings

### PR DESCRIPTION
**Pull request by ztephm**
hi @iriberri this is also from https://docs.google.com/a/cartodb.com/document/d/10WCzucZSCpmglhUF8WIU79Jo0eXJ5jcuHfxCxr4Vns4/edit?usp=sharing

...would u mind previewing? thank u!

### CODE REVIEW

File: tips-and-tricks.md [hunk 1]
```diff
@@ -38,6 +38,14 @@ Strings are any combination of numbers and characters in UTF-8. There is no limi
 
 Strings can be very useful for categorizing, grouping, or excluding data from your SQL queries. The string columns in CartoDB are case-sensitive, meaning you can use them for precision queries. There are also ways to query them for less precise comparisons of data. See the following for an example of non-case-sensitive matching of only part of a string field.
 
+If you’re editing your table’s string column by completely deleting text from one of it’s fields, the field will stay blank instead of becoming “null”. To make it “null”, you can apply this SQL query:
+
+{% highlight sql %}
+SELECT * FROM {table_name} WHERE country_name ilike 'united%'
```
Comment (iriberri) [inline tips-and-tricks.md]: This is wrong!

UPDATE table_name SET column_name = null WHERE column_name ilike ''

File: tips-and-tricks.md [hunk 2]
```diff
@@ -38,6 +38,14 @@ Strings are any combination of numbers and characters in UTF-8. There is no limi
 
 Strings can be very useful for categorizing, grouping, or excluding data from your SQL queries. The string columns in CartoDB are case-sensitive, meaning you can use them for precision queries. There are also ways to query them for less precise comparisons of data. See the following for an example of non-case-sensitive matching of only part of a string field.
 
+If you’re editing your table’s string column by completely deleting text from one of it’s fields, the field will stay blank instead of becoming “null”. To make it “null”, you can apply this SQL query:
```
Comment (iriberri) [inline tips-and-tricks.md]: dataset :)

**Comment by iriberri** [comment]
ping, please check the comments above

**Comment by csobier** [comment]
@iriberri I have updated this branch with your review comments mentioned from April. Can you confirm the SQL query example in the  #### Working with strings section is correct?  (Otherwise, I just fixed some small typos while I was in this branch).  I will confirm with Steph and merge after you sign-off.

**Comment by iriberri** [comment]
It should be:

```sql
UPDATE table_name SET column_name = null WHERE column_name ilike 'whatever you want to put as null instead'
```

**Comment by csobier** [comment]
@ztephm Steph, since this was your original issue, do you have a suggestion for the null string column name?  Otherwise, are you okay if I just use:

`UPDATE table_name SET column_name = null WHERE column_name ilike 'indicate_null_column_name_here'`

**Comment by iriberri** [comment]
Hey, notice that what we're filtering is not a column name, is a certain value.

Imagine you have a table with a lot of rows in the column "country". Imagine then that you want to get all the rows that say "Italy" and you want to make them null. Then you'd use:

```sql

UPDATE table_name SET column_name = null WHERE column_name ilike 'Italy'

````

**Comment by csobier** [comment]
Of course :-)  Will just get confirmation from @ztephm in case she has a working example...

UPDATE table_name SET column_name = null WHERE column_name ilike 'indicate_null_field_value_here'

**Comment by ztephm** [comment]
hi, i don't have a working example to link to but i can build one if we need, let me know. 

looks good, only thing that might be a little unclear to new user is "indicate_null_field_value_here", because to me that makes it sound like you're also searching for fields where the literal text 'null' occurs already instead of something like Italy. What do you think of just saying "value_you're_replacing" or something like that?

**Comment by csobier** [comment]
@ztephm Okay, I think we are getting closer :-)  I just added Carla suggestion about Italy as an example value and intro text around it.  Does this work for you?

**Comment by iriberri** [comment]
Makes more sense, thanks @csobier! Actually the first @ztephm example was using United States, but in a "hidden" way!

**Comment by ztephm** [comment]
Italy works for me :)

**Comment by csobier** [comment]
@ztephm @iriberri Fingers crossed this works for everyone?

**Comment by iriberri** [comment]
Actually the old one was super perfect for me!  This one is valid too.

:+1: for me, but let me be picky and make you think: What do you think about adding these super general details about SQL queries in such a specific question?

**Pull request merged**